src/pages/post-feed/usersDummyData.ts

```diff
@@ -0,0 +1,56 @@
+export interface User {
+  id: number;
+  username: string;
+  userRank: string;
+  userPhoto: string;
+  text: string;
+  photo: string;
+}
+
+export interface UserType {
+  id: number;
+  userPhoto: string;
+}
+
+export const posts: User[] = [
```

Comment:
```js
export const posts: User[] = [
```
if these are `posts`, the type should not be `User`, if they are `users` the name should not be `posts`